langchain/llms/llamacpp.py

```diff
@@ -167,31 +195,35 @@ def _call(self, prompt: str, stop: Optional[List[str]] = None) -> str:
         Example:
             .. code-block:: python
 
-                from langchain.llms import LlamaCppEmbeddings
-                llm = LlamaCppEmbeddings(model_path="/path/to/local/llama/model.bin")
+                from langchain.llms import LlamaCpp
+                llm = LlamaCpp(model_path="/path/to/local/llama/model.bin")
                 llm("This is a prompt.")
         """
-
-        params = self._default_params
-        if self.stop and stop is not None:
-            raise ValueError("`stop` found in both the input and default params.")
-        elif self.stop:
-            params["stop_sequences"] = self.stop
-        elif stop:
-            params["stop_sequences"] = stop
+        print("Prompt: ", prompt)
```

Comment:
Will want to remove print statements before landing

Reply:
Oops, I was testing with load_summarize_chain and forgot this was in! Fixed now.